src/state/atoms/releaseAtom.test.ts

```
@@ -0,0 +1,59 @@
+import { createStore } from 'jotai';
+import { describe, expect, it, beforeEach } from '@jest/globals';
```

ESLint

[error] Member 'beforeEach' of the import declaration should be sorted alphabetically.

(sort-imports)